src/collections/_documentation/workflow/alerts-notifications/index.md

```diff
@@ -20,16 +22,26 @@ Metric Alerts are currently in beta. For more information about access to Metric
     level="warning"
 %}
 
-Metric alerts allow you to filter and set thresholds on all errors across a project. Use metric alerts for high-level monitoring of patterns or fine-grained monitoring of individual events. Metric alerts help express lightweight service-level objectives (SLOs) such as users affected by signup-page errors or volume of database errors.
+Use Metric Alerts to:
+- Alert on service-level metrics. Is my service affecting more than X users?
+- Alert on events across issues. How many errors on the signup page?
+- Alert when a metric goes below a threshold. For example, when something stops happening.
```

Comment:
@adhiraj @billyvg Is there a more specific word or example we can use to replace "something" in this sentence? Like, when a percentage of users stop using an app?